### PR TITLE
Announce leading player changes, trump calls, and one-to-win calls

### PR DESCRIPTION
**Pull request by scottnm**

### CODE REVIEW

File: Assets/Scripts/AudioManager.cs [hunk 1]
```diff
@@ -140,6 +146,15 @@ private void RegisterEvents()
...
             {
                 mSource.PlayOneShot(mCoinDeposited);
+                int playerNumber = NetworkManager.GetPlayerNumber(PhotonPlayer.Find(ownerId));
+                if (newDepositBalance == GameConstants.AMOUNT_OF_COINS_TO_WIN - 1)
+                {
+                    mSource.PlayOneShot(mAboutToWinCalls[playerNumber]);
+                }
+                else if (newDepositBalance > ABOUT_TO_WIN_LIMIT)
+                {
+                    mSource.PlayOneShot(mTrumpingCalls[playerNumber]);
```
Comment (phoenixbishea) [inline Assets/Scripts/AudioManager.cs]: This plays the line on the 10th coin deposit. Which is undesired behaviour.

**Comment by phoenixbishea** [comment]
Do we have sound/want to setup cues for players that have more than 10 coins? So do something in the audio system oncoincollected when coins are greater than amount to win.

**Comment by scottnm** [comment]
Please take another look

**Comment by phoenixbishea** [comment]
Should the trumping call limit be a fixed value? For instance what if another player has 9 coins and you get your 8th coin. Then trumping the competition plays. This would not make sense. Is that scenario possible with the current implementation?

**Comment by phoenixbishea** [comment]
If you don't care leave it (unless you can think of another way). Another option would be to cycle trumping the competition with player _____ takes the lead.

**Pull request merged**